tests/Integration/Database/EloquentTransactionWithAfterCommitUsingRefreshDatabaseOnMultipleConnectionsTest.php

```diff
@@ -2,25 +2,14 @@
 
 namespace Illuminate\Tests\Integration\Database;
 
+use Illuminate\Support\Facades\DB;
 use Orchestra\Testbench\Attributes\WithConfig;
 
 use function Orchestra\Testbench\artisan;
 
 #[WithConfig('database.connections.second', ['driver' => 'sqlite', 'database' => ':memory:', 'foreign_key_constraints' => false])]
 class EloquentTransactionWithAfterCommitUsingRefreshDatabaseOnMultipleConnectionsTest extends EloquentTransactionWithAfterCommitUsingRefreshDatabaseTest
 {
-    /** {@inheritDoc} */
-    protected function setUp(): void
-    {
-        $this->afterApplicationCreated(function () {
-            $this->markTestSkippedWhen(
-                $this->usesSqliteInMemoryDatabaseConnection(), 'Skipped when default database is configured to use In-Memory SQLite Database'
-            );
-        });
-
-        parent::setUp();
-    }
```

Comment:
I added this since I believe the default connection and `second` would refer to the same "in-memory" database, but I guess it does matter much as it does refer to 2 separate PDO instance.

Reply:
Yep. There's actually an issue with in-memory database connections with multiple databases in 11.x right now, I've sent a [PR to fix it on 12.x](https://github.com/laravel/framework/pull/53231) (API changed), but the fact that it's an in-memory or disk-based database connection shouldn't matter for the tests, IMO.